src/buffer/out/textBuffer.cpp

```diff
@@ -944,6 +944,115 @@ Microsoft::Console::Render::IRenderTarget& TextBuffer::GetRenderTarget() noexcep
     return _renderTarget;
 }
 
+// Method Description:
+// - Get the COORD for the beginning of the word you are on
+// Arguments:
+// - target - a COORD on the word you are currently on
+// - wordDelimiters - what characters are we considering for the separation of words
+// - includeCharacterRun - include the character run located at the beginning of the word
+// Return Value:
+// - The COORD for the first character on the "word"  (inclusive)
+const COORD TextBuffer::GetWordStart(const COORD target, const std::wstring wordDelimiters, bool includeCharacterRun) const
+{
+    COORD result = target;
+
+    // can't expand left
+    if (target.X == GetSize().Left())
+    {
+        return result;
+    }
+
+    auto bufferIterator = GetTextDataAt(result);
+    const auto startedOnDelimiter = _GetDelimiterClass(*bufferIterator, wordDelimiters);

```

Comment:
```suggestion
    const auto initialDelimiter = _GetDelimiterClass(*bufferIterator, wordDelimiters);
```

`startedOnDelimiter` makes it sound like a bool - "did I start on a delimiter?"

Reply:
It actually used to be a bool haha. Good catch.